base/vulkanexamplebase.cpp

```diff
@@ -2145,18 +2145,20 @@ void VulkanExampleBase::setupRenderPass()
 
 	dependencies[0].srcSubpass = VK_SUBPASS_EXTERNAL;
 	dependencies[0].dstSubpass = 0;
-	dependencies[0].srcStageMask = VK_PIPELINE_STAGE_BOTTOM_OF_PIPE_BIT;
+	// cannot be much sure here what the previous work is
+	// assumably it is the semaphore to vkQueueSubmit, otherwisely it is the user's resposibility to otherwisely synchronize
+	dependencies[0].srcStageMask = this->submitPipelineStages;
```

Comment:
Too many examples rely on this, so I am not 110 % sure what the correct stage should be here...